src/installer/tests/HostActivation.Tests/DependencyResolution/RidAssetResolution.cs

```diff
@@ -139,6 +139,7 @@ public void RidSpecificAssembly_RidGraph(string rid, string includedPath, string
                 excludedPath);
         }
 
+        [SkipOnPlatform(TestPlatforms.OSX, "https://github.com/dotnet/runtime/issues/91486")]
```

Comment:
I'm missing why the two tests in here are affected by this issue. They don't use `TestOnlyProductBehavior`?